lib/ui/view/all_threads_view/threadDetail/hook_thread_detail.dart

```diff
@@ -0,0 +1,442 @@
+                                              Icons.attach_file_outlined,
+                                              color: Color(0xFF424141),
+                                            )),
+                                      ],
+                                    ),
+                                    IconButton(
+                                        onPressed: () {
+                                          if (!_messageController.text
+                                              .toString()
+                                              .isEmpty) {
+                                            viewmodel.addReply(
+                                                new ThreadReplies(
+                                                    imageUrl: viewmodel
+                                                        .data.userImageUrl,
+                                                    message:
+                                                        _messageController.text,
+                                                    senderName:
+                                                        viewmodel.data.userName,
+                                                    time: viewmodel.time()));
+                                            _messageController.text = "";
+                                            FocusScope.of(context)
+                                                .requestFocus(FocusNode());
+                                            _scrollController.jumpTo(
+                                                _scrollController
+                                                    .position.maxScrollExtent);
+                                          }
+
+                                         
+                                        },
+                                        icon: Icon(
+                                          Icons.send,
+                                          color: Color(0xFFBEBEBE),
+                                        ))
+                                  ],
+                                ))
+                          ],
+                        ),
+                      ),
+                    )
+                  ]),
+            )));
+  }
+
+  void showModalSheet(BuildContext context) {
+    showModalBottomSheet(
+        elevation: 10,
+        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(15)),
+        context: context,
+        builder: (context) {
+          return Container(
+            child: Expanded(
+              child: Column(
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.all(12.0),
+                    child: Row(
+                      children: [
+                        Icon(Icons.access_time_rounded,
+                            color: AppColors.borderColor),
+                        SizedBox(width: 30),
+                        CustomText(text: "Remind Me", fontSize: 15)
```

Comment:
use one of the styles in styles.dart or add yours if it's not there..... DO THIS FOR ALL SIMILAR ISSUES

Reply:
Okay